weekly-mission/next-project/src/components/CardList/index.tsx

```diff
@@ -1,22 +1,49 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import SearchBar from "@components/Searchbar";
 import Folder from "../Folder";
-import useFolderList from "pages/service/useFolderList";
 import { memo } from "react";
 import Card from "../Card";
 import styles from "./CardList.module.css";
+import instance from "lib/api";
+import { useRouter } from "next/router";
+interface Link {
+  id: number;
+  created_at: string;
+  updated_at: string | null;
+  url: string;
+  title: string;
+  description: string;
+  image_source: string;
+  folder_id: number | null;
+}
 
 interface CardListProps {
   isFolderPage: boolean;
+  userId?: number | null;
+  folderId?: string | string[] | null | undefined;
 }
 
-const CardList = ({ isFolderPage }: CardListProps) => {
-  const { data: folderData, isLoading } = useFolderList();
+const CardList = ({ isFolderPage, userId, folderId }: CardListProps) => {
+  const [links, setLinks] = useState<Link[]>([]);
+
+  async function getSharedLinks() {
+    if (!userId) {
+      return;
+    }
+
+    let url = `/users/${userId}/links`;
+    if (folderId) {
+      url += `?folderId=${folderId}`;
+    }
```

Comment:
```suggestion
   const url = `/users/${userId}/links` + (folderId ? `?folderId=${folderId}` : '');
```
const 로 쓸 수 있는 것들은 최대한 const 로 써주면 좋습니다!